.splunk-version

```diff
@@ -0,0 +1,2 @@
+8.0.5
```

Comment:
Straight copy of https://github.com/openshift/splunk-forwarder-operator/blob/c28e0fb044330d413e18cd7f0a43ab1673d2aa18/.splunk-version